.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,14 +4,14 @@ about: Create a report to help us improve
 title: ''
 labels: ''
 assignees: ''
-
```

Comment:
There are a few changes to `.md` files here that I believe are from prettier running after a merge commit.